environment.yml

```diff
@@ -54,4 +54,4 @@ dependencies:
       - DFO-LS
       - fides==0.7.4
       - gif
-      - gif[matplotlib]
```

Comment:
I guess in the dev environment we still need to keep seaborn and gif[matplotlib] to generate some of the plots in the documentation. Can you confirm by running pytask once in the docs folder?

Reply:
You're right. For the documentation we need both seaborn and matplotlib.